docs/layers/grid-layer.md

```diff
@@ -5,19 +5,21 @@
   <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="extruded" />
 </p>
 
-# GridLayer
+# Grid Layer
```

Comment:
I took the arc layer as reference, but it might have been this one that was out of sync